src/classes/DataFactoryTest.cls

```diff
@@ -15,5 +15,15 @@ public class DataFactoryTest {
         Account a = (Account) adf.build();
```

Comment:
This variable doesn't seem to have a very meaningful name. I suggest to give this variable a concise but meaningful name that suggest its purpose.